### ISSUE TITLE
Investigate completely removing NSPR

### ISSUE THREAD

**Issue by mreynolds389** [389ds/389-ds-base#4452]
**Issue Description**

Look into removing all the NSPR types and functions and only using pthreads and fixed width types.

**Comment by Firstyear** [comment]
Agreed this would be good. Probably a long term goal where we clean up in small batches over time.
